FreeTubeApp/FreeTube
# Update copy/open YT video in playlist view to have playlist ID when present

# Update copy/open YT video in playlist view to have playlist ID when present




## Pull Request Type


## Related issue



Similar to #1258

## Description
Same as title

## Screenshots 
![image](https://user-images.githubusercontent.com/1018543/200764212-a60c74bb-9023-4cec-b6b8-4bbb84c4eef1.png)
![image](https://user-images.githubusercontent.com/1018543/200764241-84237645-4b16-48e0-a3d6-e3eaa09cd6cf.png)



## Testing 



- Open a playlist
- Copy video URL/open video
- Should have list

Only playlist view and playlist in video playing view are tested by me, please test more views~

## Desktop

- **OS:** MacOS
- **OS Version:** 12.6.1
- **FreeTube version:** 2b233815ec7a8a62abebec92f791ca355110bbe6

## Additional context
I found that this feature is absent when I try to copy a video URL with playlist for non current playing video
